go.mod

```diff
@@ -254,6 +254,7 @@ require (
 )
 
 require (
+	cloud.google.com/go/kms v1.0.0
```

Comment:
I've seen [there's a `v1.1.0`](https://pkg.go.dev/cloud.google.com/go/kms@v1.1.0/apiv1?tab=versions), so I'm wondering if we should use the most up to date version of the library. 

What do you think? 🤔

Reply:
yeah, I can change it